packages/components/src/pages/MultiPage.vue

```diff
@@ -68,8 +68,12 @@ export default {
     },
 
     next() {
-      const nextId = this.getPageId(1);
-      this.jumpTo(nextId);
+      if (this.currentPage === 'record-appmaps') {
```

Comment:
Is this handling a legacy condition? What if the current page is "runtime analysis"?

Reply:
There is no `runtime analysis` page anymore, so it can't be `runtime-analysis` once this change is merged.